github_connector/wizards/wizard_create_team.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2017-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class WizardCreateTeam(models.TransientModel):
+    _name = 'wizard.create.team'
+    _inherit = ['github.team']
+
+    # Overload Columns Section
+    name = fields.Char(readonly=False)
+    description = fields.Char(readonly=False)
+    organization_id = fields.Many2one(readonly=False)
+    privacy = fields.Selection(readonly=False)
+
+    # Columns Section
+    wizard_partner_ids = fields.Many2many(
+        string='Members', comodel_name='res.partner',
+        domain="[('github_login', '!=', False)]")
+
+    wizard_repository_ids = fields.Many2many(
+        string='Repositories', comodel_name='github.repository')
```

Comment:
I get this warning:
`Two fields (repository_ids, wizard_repository_ids) of wizard.create.team() have the same label: Repositories.`
You can change string `Repositories` by, i.e., `Repositories Team` here or in github.team class.